domains/midnight.localplayer.dev.json

```diff
@@ -0,0 +1,17 @@
+{
+  "description": "My website for my personal hobbies.",
+
+  "domain": "localplayer.dev",
+  "subdomain": "midnight",
+
+  "owner": {
+    "email": "kankeet56@outlook.com"
+  },
+
+  "record": {
+    "CNAME": ["013c5987-dd17-459e-9cbc-d94ae0540f0e.id.repl.co"],
```

Comment:
CNAME records cannot be used in conjunction with other records and vise versa.

Reply:
oh okay!